cli/npm/registry.rs

```diff
@@ -98,7 +99,22 @@ pub struct NpmRegistryApi {
 
 impl NpmRegistryApi {
   pub fn default_url() -> Url {
-    Url::parse("https://registry.npmjs.org").unwrap()
+    let env_var_name = "DENO_NPM_REGISTRY";
+    if let Ok(registry_url) = std::env::var(env_var_name) {
```

Comment:
Not for this PR but I feel like we should load all recognized env vars upfront and put it in the config, WDYT @dsherret?

Reply:
Yeah, that would be more desirable, but also probably in the future we'll want to support multiple registries at the same time instead of just one.